multimedia/dvdauthor/Portfile

```diff
@@ -34,6 +34,7 @@ depends_lib         port:libdvdread \
                     port:zlib \
                     port:libpng \
                     port:libiconv \
+                    port:icu \
```

Comment:
Rather than add the icu dependency, it would be better to apply this patch so that dvdauthor uses pkg-config instead of xml2-config to find libxml2, thereby avoiding the link to the indirect icu dependency:

https://github.com/ldo/dvdauthor/pull/30